client/platform/web-girder/api/viame.service.ts

```diff
@@ -46,7 +46,7 @@ function makeViameFolder({
 }: {
   folderId: string;
   name: string;
-  fps: number;
+  fps: number | string;
```

Comment:
may be a remnant before the changing of other items.  Only use of the function does a `parseInt` on the value before passing it.

Reply:
I have no recollection of this change.  I'll remove it.